src/main/java/com/revealprecision/revealserver/service/FormFieldService.java

```diff
@@ -0,0 +1,27 @@
+package com.revealprecision.revealserver.service;
+
+import com.revealprecision.revealserver.api.v1.dto.factory.FormFieldFactory;
+import com.revealprecision.revealserver.api.v1.dto.request.FormFieldRequest;
+import com.revealprecision.revealserver.enums.EntityStatus;
+import com.revealprecision.revealserver.persistence.domain.FormField;
+import com.revealprecision.revealserver.persistence.repository.FormFieldRepository;
+import lombok.RequiredArgsConstructor;
+import org.springframework.stereotype.Service;
+
+@Service
+@RequiredArgsConstructor
+public class FormFieldService {
+
+  private final FormFieldRepository formFieldRepository;
+
+  public FormField findByNameAndFormTitle(String name, String formTitle) {
+    return formFieldRepository.findByNameAndFormTitle(name, formTitle);
+  }
+
+  public FormField createFormField(FormFieldRequest formFieldRequest) {
```

Comment:
is it that uniqueness on (name, form_field) is handled at database level, but that failure is only visible on debug, maybe we throw not unique exception at api level.